### PR TITLE
Add translations

### LINKED ISSUE #12: Add i18n support
We will use https://lingui.js.org/ package.  Language should be taken from:

1. url,
2. localstorage,
3. navigator.language

### LINKED ISSUE #314: Translate literals
The storefront doesn't have any code to handle translations. The literals are hardcoded in English, like the image below.

Is there any movement to translate it? I think the approach used in the dashboard-next could be used here as well.

I can start that using the same structure used in dashboard-next if there isn't issues.

image

### PR DESCRIPTION
**Pull request by AlicjaSzu**
I want to merge this change because...

resolves #12 
resolves  #314

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] Changes are mentioned in the changelog.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -81,19 +87,20 @@
     "@types/storybook__react": "^4.0.2",
     "@types/styled-components": "^4.1.16",
     "apollo-storybook-react": "^0.2.1",
+    "babel-core": "^7.0.0-bridge.0",
```
Comment (jwm0) [inline package.json]: We are already using babel core as @babel/core

File: src/translations/loadMessagesCatalog.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  (
+    (navigator.languages && navigator.languages[0]) ||
+    navigator.language ||
+    fallbackLang
+  ).toLowerCase()
+);
+
+export const loadCatalogs = async (language: string = lang) => {
+  const catalog = await import(
+     /* webpackMode: "lazy", webpackChunkName: "i18n-[index]" */
```
Comment (mateuszkula) [inline src/translations/loadMessagesCatalog.tsx]: Do we need this comment?

Comment (bogdal) [reply src/translations/loadMessagesCatalog.tsx]: As [the doc](https://lingui.js.org/guides/dynamic-loading-catalogs.html) says it's webpack’s `magic` comment.

File: src/locales/_build/src/components/MainMenu/MainMenu.tsx.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (bogdal) [inline src/locales/_build/src/components/MainMenu/MainMenu.tsx.json]: We should get rid of this file and add `src/locales/_build` to the `.gitignore` file.

File: .linguirc [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+   "localeDir": "src/locales/",
+   "srcPathDirs": ["src/"],
+   "format": "po",
+   "sourceLocale": "en",
+   "extractBabelOptions": {
+      "plugins": ["@babel/plugin-syntax-dynamic-import", "macros"]
```
Comment (bogdal) [inline .linguirc]:
```suggestion
      "plugins": ["@babel/plugin-syntax-dynamic-import"]
```

**Pull request merged**